config/test.json

```diff
@@ -0,0 +1,3 @@
+{
+    "logging": "null"
```

Comment:
@philbooth thoughts on the name `null` ?

Reply:
I like it.